advanced/Scripts/piholeLogFlush.sh

```diff
@@ -16,7 +16,8 @@ source ${colfile}
 # Constructed to return nothing when
 # a) the setting is not present in the config file, or
 # b) the setting is commented out (e.g. "#DBFILE=...")
-DBFILE="$(sed -n -e 's/^\s^.DBFILE\s*=\s*//p' /etc/pihole/pihole-FTL.conf)"
+FTLconf="/etc/pihole/pihole-FTL.conf"
+[ -e "$FTLconf" ] && DBFILE="$(sed -n -e 's/^\s*DBFILE\s*=\s*//p' $FTLconf)"
```

Comment:
Just a stylistic issue, not a functional change, but can you set the variable expansion to `${FTLconf}`?

Reply:
Done in a new commit.